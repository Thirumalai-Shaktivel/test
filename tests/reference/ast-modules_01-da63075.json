{
    "basename": "ast-modules_01-da63075",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/modules_01.f90",
    "infile_hash": "17c761eb72726e0dde59f2b21f98f2402d783613834e927c49047534",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-modules_01-da63075.stdout",
    "stdout_hash": "23397cbd7887123e11d7ed329233094e1ba128408242006613e6ed99",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}