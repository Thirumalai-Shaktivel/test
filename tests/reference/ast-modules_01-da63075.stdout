(TranslationUnit [(Module a_01 [] [(ImplicitNone [])] [] [(Subroutine b [] [] () [] [] [] [] [(Print 0 () [(Str "b()")] (TriviaNode [] [(EmptyLines)]))] [])]) (Program modules_01 [(Use [] a_01 [(UseSymbol b ())])] [(ImplicitNone [])] [] [(SubroutineCall 0 b [] [] [])] [])])
